.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2.1
+executors:
+  node-executor:
+    docker:
+      - image: circleci/node:10.15
+    working_directory: ~/keystone
+
+commands:
+  set-npm-auth:
+    description: "Set NPM authentication"
+    steps:
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/repo/.npmrc
+
+  install-deps:
+    description: "Install build dependencies"
+    steps:
+      - restore_cache:
+          keys:
+            - v1-dependencies-build-{{ checksum "yarn.lock" }}
+      - run: yarn install --forzen-lockfile
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-build-{{ checksum "yarn.lcok" }}
+
+jobs:
+  build:
+     executor: node-executor
+     steps:
+       - checkout
+       - install-deps
+       - run: yarn build
+       - persist_to_workspace:
+          root: ~/keystone
+          paths:
+            - ./*
+  publish:
+    executor: node-executor
+    working_directory: ~/keystone
+    steps:
+      - attach_workspace:
+          at: .
+      - set-npm-auth
+      - run: npm publish
+
+workflows:
+  version: 2.1
+  build-and-publish:
+    jobs:
+      - build
+      - publish:
+          context: twreporter
+          requires:
+            - build
+          filters:
+            tags:
```

Comment:
Q: question about the design decision: why do we choose to publish according to tagged target commit rather than publish every time when commits push to master ?

Reply:
RC is a release candidate. In the current scenario, we are trying to use the trunk-based branching strategy to publish our package. If we want to publish in every change commit, then every commit should update the version field of package.json. Also, each release candidate consists of a single change. Is that what you prefer?

Reply:
No, current solution looks good to me.